src/Http/routes.php

```diff
@@ -6,6 +6,10 @@
     Route::resource('Patient', '\LibreEHR\FHIR\Http\Controllers\PatientController');
     Route::resource('Patients', '\LibreEHR\FHIR\Http\Controllers\PatientController');
 
+    Route::resource('get_schedule', '\LibreEHR\FHIR\Http\Controllers\AppointmentController@getSchedule');
```

Comment:
It seems like you're getting the hang of the routing, but the routes need to follow the FHIR spec for the Appointment, Slit and Schedule resources. Instead of get_schedule this should be Schedule if it applies to the schedule resource, and point to a ScheduleController.